src/views/Feed/FeedExtra.js

```diff
@@ -0,0 +1,62 @@
+import _ from 'lodash'
+import cx from 'classnames'
+import React, { PropTypes } from 'react'
+
+import META from '../../utils/Meta'
+import { customPropTypes, getUnhandledProps, useKeyOnly } from '../../utils/propUtils'
+
+function FeedExtra(props) {
+  const { children, className, images, text } = props
+  const classes = cx(
+    className,
+    useKeyOnly(images, 'images'),
+    useKeyOnly(text, 'text'),
+    'extra'
+  )
+  const rest = getUnhandledProps(FeedExtra, props)
+
+  if (Array.isArray(images)) {
+    const imagesJSX = images.map((image, i) => {
+      return _.isString(image) ? <img key={i} src={image} /> : image
```

Comment:
@levithomason It seems that you like antipattern with `key` more than I :laughing: 

I think:
- allow pass only string, not nodes;
- use pair of `index` and `src`.

Better ideas?